test/unit/gateways/card_stream_test.rb

```diff
@@ -33,7 +33,7 @@ def setup
       three_d_secure: {
         enrolled: 'true',
         authentication_response_status: 'Y',
-        eci: 05,
+        eci: '05',
```

Comment:
I thought this was previously changed to this for some reason? was Rubocop complaining about it?

Reply:
yup @curiousepic  I change due a Rubocop complain.